src/main/java/dev/codesupport/web/common/security/access/Accessor.java

```diff
@@ -0,0 +1,9 @@
+package dev.codesupport.web.common.security.access;
+
+/**
+ * Used for access evaluations with no specific associated class type
+ */
+public enum Accessor {
+    NONE,
+    DISCORD
+}
```

Comment:
I gather these are just assigned automatically to `0` and `1`?

Reply:
something like that, they aren't going to be used in the database though, they are only used in internal logic